rollup.config.js

```diff
@@ -7,7 +7,7 @@ export default {
   input: './index.ts',
   output: [
     {
-      file: pkg.main,
+      file: pkg.exports['.'].require,
```

Comment:
:information_source: This fixes the override of the `index.js` file in the `dist` folder